FrameworkSystem/DB/InstalledComponentsDB.py

```diff
@@ -429,10 +429,10 @@ def __filterFields(self, session, table, matchFields=None):
         actualKey = key.replace('.smaller', '')
 
       if matchFields[key] is None:
-        sql = '%s IS NULL' % (actualKey)
+        sql = '`%s` IS NULL' % (actualKey)
       elif isinstance(matchFields[key], list):
         if len(matchFields[key]) > 0 and None not in matchFields[key]:
-          sql = '%s IN ( ' % (actualKey)
+          sql = '`%s` IN ( ' % (actualKey)
```

Comment:
Aren't we better off just moving all these trivial DBs to SQLAlchemy once for all ?

Reply:
...this specific DB is already written in SQLAlchemy!

Reply:
but why do we have to manipulate query string then ?

Reply:
MySQL v8 introduces several 'new' reserved word (`system` for instance). These can be used as column-name only if back-tick-quoted in queries. Quoting every column-name is maybe too much, but safe. Anyway, column-names (and table-names) were already quoted in the `SELECT` and `FROM` clauses. They weren't in the `WHERE` one.

Reply:
This i understand. I am just wondering why in a SqlAlchemy powered DB, we need to write queries at all ? SQLAlechemy always escapes column names

Reply:
The strange thing is that in this DB there's one method where an sql statement is constructed. This is definitely not ideal. We can think or re-write the `__filterFields` method or accept this PR as-is and maybe re-write later.